GW Launcher/Account.cs

```diff
@@ -15,11 +20,11 @@ public class Account
     public string character = "";
 
     [JsonRequired]
-    public string gwpath = "";
+    public string path = "";

```

Comment:
do not rename this as it will break compatibility with old Account.json files
also the name is better left alone